embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -0,0 +1,38 @@
+package io.embrace.android.embracesdk.arch.schema
+
+internal object EmbType {
```

Comment:
We want the name to be lower-case in the payload - I can't find a way to override `name`, so I used `typeName` before. Is there a way to do that so the default enum name would just work?

Reply:
I'm not sure I follow. This class doesn't serialize/deserialize JSON, it just holds data that is passed to other classes that do.